articles/service-fabric/service-fabric-get-started-mac.md

```diff
@@ -64,34 +58,58 @@ To set up a local Docker container and have a Service Fabric cluster running on
     >
     >The recommended approach is to directly modify the daemon configuration settings in Docker. Select the **Docker icon**, and then select **Preferences** > **Daemon** > **Advanced**.
     >
+    >We recommend increasing the resources allocated to docker when testing large complex applications, this can be changed by selecting **Docker icon**, and then select **Advanced** then adjusting the number of cores and memory as required.
```

Comment:
Split this into two sentences. 

We recommend increasing the resources allocated to Docker when testing large applications. This can be done by selecting the **Docker Icon**, then selecting **Advanced** to adjust the number of cores and memory.